register_commands/register.js

```diff
@@ -8,12 +8,56 @@ const headers = {
   "Content-Type": "application/json"
 }
 
+let commands = []
+
 let command_data = {
   "name": "foo",
   "type": 1,
   "description": "replies with bar ;/",
 }
 
-axios.post(url, JSON.stringify(command_data), {
-  headers: headers,
-})
+commands.push(command_data)
+
+let command_data_with_options = {
+  "name": "blep",
+  "type": 1,
+  "description": "Send a random adorable animal photo",
+  "options": [
+      {
+          "name": "animal",
+          "description": "The type of animal",
+          "type": 3,
+          "required": true,
+          "choices": [
+              {
+                  "name": "Dog",
+                  "value": "animal_dog"
+              },
+              {
+                  "name": "Cat",
+                  "value": "animal_cat"
+              },
+              {
+                  "name": "Penguin",
+                  "value": "animal_penguin"
+              }
+          ]
+      },
+      {
+          "name": "only_smol",
+          "description": "Whether to show only baby animals",
+          "type": 5,
+          "required": false
+      }
+  ]
+}
+
+commands.push(command_data_with_options)
+
+commands.forEach(commmand => {
```

Comment:
there is a different Discord endpoint to overwrite all existing commands in a bulk (PUT method). if we will update multiple commands this way, we don't provide any capability to remove old commands (only new ones will be added).